### ISSUE TITLE
Implementation of comparison models

### ISSUE THREAD

**Issue by ahmadikalkhorani** [Audio-WestlakeU/FS-EEND#15]
Can you please share your implementation of "EEND-EDA+FLEX-STB" and "Offline EEND-EDA" to replicate Table 2 in the paper (https://arxiv.org/pdf/2309.13916)? 

Thanks,
Vahid

**Comment by DiLiangWU** [comment]
Certainly, we will share the code for "EEND-EDA+FLEX-STB" and "Offline EEND-EDA" once it has been properly reorganized and finalized.
